snap/info.go

```diff
@@ -41,6 +41,33 @@ import (
 	"github.com/snapcore/snapd/timeout"
 )
 
+// ContainerPlaceInfo offers all the information about where a container (which
+// can be a snap or a component) and its data are located and exposed in the
+// filesystem.
```

Comment:
```suggestion
// ContainerPlaceInfo contains all the information about where a container (which
// can be a snap or a component) and its data are located and exposed in the
// filesystem.
```

Reply:
Well, it is an interface, so it is more offering than containing. Also, the comment was copied from `type PlaceInfo`.